entrypoint.sh

```diff
@@ -44,6 +44,7 @@ elif [ -n "$POSTGRES_PORT_5432_TCP" ]; then
 		echo >&2 "  Connecting to DB_HOST ($DB_HOST)"
 		echo >&2 "  instead of the linked postgres container."
 	fi
+	: ${DB_NAME:='postgres'}
```

Comment:
I note now that drush creates a missing database, though I posit that it isn't logical.

```
db_1  | FATAL:  database "drupal" does not exist
...
web_1 | You are about to CREATE the 'drupal' database. Do you want to continue? (y/n): y
```


whatever, I give up.